docs/Makefile

```diff
@@ -92,7 +92,3 @@ html: install_build_deps
 	python -m robot.libdoc genie.libs.robot.GenieRobotApis $(BUILDDIR)/html/userguide/robot_api.html
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
-serve:
```

Comment:
no harm to have?

Reply:
its a duplicate entry

Reply:
not sure duplicate with what. I think this is just launching http server for ease to check.

Reply:
https://github.com/CiscoTestAutomation/genie/blob/fc61ef0bec6d5a55b9b3bb480bec133a58915229/docs/Makefile#L78

Reply:
![image](https://user-images.githubusercontent.com/10467987/128193815-ce45871d-cb65-419c-9ef3-135d99d04554.png)

Reply:
I didn't notice that. ok. undersood...